keps/sig-cli/1440-kubectl-events/README.md

```diff
@@ -138,17 +143,41 @@ Additionally, the new command should support all the printing flags available in
 
 ### Test Plan
 
-In addition to standard unit tests for kubectl, the events command will be released as a kubectl alpha subcommand, signaling users to expect instability. During the alpha phase we will gather feedback from users that we expect will improve the design of debug and identify the Critical User Journeys we should test prior to Alpha -> Beta graduation.
+[x] I/we understand the owners of the involved components may require updates to
+existing tests to make this code solid enough prior to committing the changes necessary
+to implement this enhancement.
+
+##### Prerequisite testing updates
+
+Before any additional functional updates we need to ensure the current functionality
+is properly cover with unit and integration (`test/cmd`) tests.
+Before promoting to beta at least a single e2e test should also be added in
+`k8s.io/kubernetes/test/e2e/kubectl/kubectl.go`.
+
+##### Unit tests
+
+- `k8s.io/kubectl/pkg/cmd/events`: `2022-09-21` - `36.6%`
+
+##### Integration tests
+
+- `k8s.io/kubernetes/test/cmd/events.sh`: [test-cmd.run_kubectl_events_tests](https://testgrid.k8s.io/sig-release-master-blocking#integration-master)
+
+##### e2e tests
+
+- missing
 
 ### Graduation Criteria
 
 Once the experimental kubectl events command is implemented, this can be rolled out in multiple phases.
 
 ##### Beta
-- [ ] Gather the feedback, which will help improve the command
-- [ ] Extend with the new features based on feedback
+
+- [x] Add e2e tests, increase unit coverage.
```

Comment:
I don't see e2e tests done?

I'm not going to block on it, because I see the test/cmd/events.sh which are already integration tests, but according to line 155, I don't see those e2e tests yet. Am I missing something?

Reply:
You are right @wojtek-t . e2e tests will be handled with this PR https://github.com/kubernetes/kubernetes/pull/111855

Reply:
Great - thanks!

@soltysh - can you please either merge that PR first or unmark this bit as not yet done?

Reply:
@wojtek-t tagged the e2e PR, so we should be go as is.

Reply:
Thanks!

And thanks for your patience and understanding in 1.25 :)